src/components/Filter/FilterModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import styles from './filter.module.css';
+
+const FilterModal: FC = () => {
+  return (
+    <div className={styles.modalContainer}>
```

Comment:
use dialog html tag for modal

Reply:
I dont think this will work here because these are modals that open in the center like prompt windows.

Reply:
No we used dialog for modal